server/src/com/cloud/network/element/VirtualRouterElement.java

```diff
@@ -171,6 +171,7 @@
     @Inject
     protected RouterDeploymentDefinitionBuilder routerDeploymentDefinitionBuilder;
 
+    static final ConfigKey<Integer> port= new ConfigKey<Integer>("Network",Integer.class, "network.loadbalancer.haproxy.stats.port", "8081", "Load Balancer(haproxy) stats port number.", true);
```

Comment:
This key is already present in the Config enum class, which is legacy.
You might consider migrating the usages of the legacy enum value, to your new one.
Also that might mean, that you have to move it in VirtualRouterElementService interface
to make sure the plugins can use it.